src/state/performance/performance.ts

```diff
@@ -23,6 +24,12 @@ interface PerformanceTrackingState {
   };
 }
 
+export function isEnabled() {
+  const isHardwareWallet = store.getState().wallets.selected?.deviceId;
```

Comment:
Just checking you're 100% sure that store.getState().wallets is never null or undefined

Reply:
good point I'll null guard it

Reply:
Actually looking at the type definition, it should be always defined. `wallets.wallets` could be `null`